storage/sqlite/store.go

```diff
@@ -51,6 +52,27 @@ type Store struct {
 	snapshotNames []string
 }
 
+func (s *Store) RollbackToBlockHeight(height uint64) error {
+	tx, err := s.db.BeginTx(context.Background(), &sql.TxOptions{})
```

Comment:
maybe missing validation if height is bigger than last height?

Reply:
sure I will add